lib/internal/navigator.js

```diff
@@ -24,6 +30,8 @@ class Navigator {
   // Private properties are used to avoid brand validations.
   #availableParallelism;
   #userAgent = `Node.js/${nodeVersion.slice(1, nodeVersion.indexOf('.'))}`;
```

Comment:
```suggestion
  #userAgent = `Node.js/${StringPrototypeSlice(nodeVersion, 1, StringPrototypeIndexOf(nodeVersion, '.'))}`;
```
as long as you're in there?

Reply:
@ljharb 

Please no. :)))

I know you want to change it, but that is why i created #50467 . And you know how much discussions it created. I think it has now enough approvals and you could merge it.

Reply:
ah gotcha, thanks, pls resolve this then :-)